foundation/plugins/org.jboss.tools.foundation.help.ui/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/jboss-dmr-1.2.0.Final.jar"/>
```

Comment:
if necessary we should be able to add all three to locus. 
But at least we can avoid duplication for the basic act of parsing json.